src/Controller/DataFlowJobController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of itk-dev/datatidy.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace App\Controller;
+
+use App\Entity\DataFlowJob;
+use App\Repository\DataFlowJobRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/data/flow2/job")
```

Comment:
Why `2`? A typo? A feature?

Reply:
It clashed with another route. Fixed it by changing the route for dataflowjobs.